labgrid/driver/dediprogflashdriver.py

```diff
@@ -0,0 +1,69 @@
+import os.path
+import logging
+import attr
+
+from ..resource import NetworkDediprogFlasher
+from ..factory import target_factory
+from ..step import step
+from .common import Driver, check_file
+from ..util.managedfile import ManagedFile
+from ..util.helper import processwrapper
+
+
+@target_factory.reg_driver
+@attr.s(eq=False)
+class DediprogFlashDriver(Driver):
+    """ The DediprogFlashDriver uses the dediprog utility to write an image
+    to a raw rom. The driver is a pure wrapper of the dpcmd utility"""
+    bindings = {
+        'flasher': {"DediprogFlasher", NetworkDediprogFlasher},
+    }
+
+    image = attr.ib(validator=attr.validators.instance_of(str), default='')
```

Comment:
```suggestion
    image = attr.ib(validator=attr.validators.optional(attr.validators.instance_of(str)), default=None)
```
Oh, I'm sorry that my suggestion did not work. That's what I meant, actually.

Reply:
Fixed it myself, so we can get this merged.